docs/source/datasets/index.rst

```diff
@@ -4,9 +4,13 @@
 Datasets
 ========
 
-Here we cover the realistic simulated datasets, which are analogous to 'real world' administrative records such as tax documents
-and census surveys, that users can generate using :code:`pseudopeople` for developing and testing Entity Resolution algorithms 
-and software.
+Here we cover the realistic simulated datasets, which are analogous to "real world" administrative records such as tax documents
+and census surveys, that users can generate using Pseudopeople for developing and testing Entity Resolution algorithms 
```

Comment:
```suggestion
and routinely generated files of social security numbers, that users can generate using Pseudopeople for developing and testing Entity Resolution algorithms 
```
and did we make a decision about `pseudopeople` vs pseudopeople vs Pseudopeople?  I think we are headed for lowercase and not fixed width.